include/tvm/solver/internal/Option.h

```diff
@@ -16,7 +16,7 @@ public:                                                                    \
   }
 
 /** Adding an option \a optionName of type \a type with no default value.
- * (The default value of the underlying solver will be used.
+ * The default value of the underlying solver will be used.
  */
 #define TVM_ADD_DEFAULT_OPTION(optionName, type) \
```

Comment:
This is not related to this PR but why do `TVM_ADD_NON_DEFAULT_OPTION` still store things as `std::optional<T>` when there's no way for them not to hold a value?